web_responsive/readme/ROADMAP.rst

```diff
@@ -4,3 +4,5 @@
   new size. This is Odoo's own limitation.
 * App navigation with keyboard.
 * Make it more beautiful. Maybe OCA-branded?
+* The button for manage list view columns dissapears when scroll because
+  can't have sticky position.
```

Comment:
You can do it :wink::

1. Override https://github.com/odoo/odoo/blob/21cccf9d7235b80fab30e19ce3fbd5df4a8ee686/addons/web/static/src/js/views/list/list_renderer.js#L916
1. Check if `.o_optional_columns_dropdown_toggle` exists.
1. If so, move it to the end of the last `thead>tr[0]>th` element.

The rest is playing a bit with CSS, and you're done.

You'd have to check what happens if the last col is hidden, though. Maybe you'd have to append it directly to the end of `thead>tr`.